ea_restaurant_data_loader/test/lib/utils/oauth2_util_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule EaRestaurantDataLoader.Oauth2UtilTest do
+  use ExUnit.Case
+  doctest EaRestaurantDataLoader.Oauth2Util
+  alias EaRestaurantDataLoader.Oauth2Util
+
+  describe "Oauth2Util Test" do
+    test "build client credential token successfully" do
+      token = Oauth2Util.build_client_credentials_token("ea_restaurant", "READ", 1, "1234")
+      {_, claims} = Oauth2Util.get_token_decoded(token, "1234")
+      assert "ea_restaurant" == claims["clientName"]
+      assert "READ" == claims["scopes"]
+    end
+
+    test " verify expired token" do
+      token = Oauth2Util.build_client_credentials_token("ea_restaurant", "READ", -1, "1234")
+      {_, claims} = Oauth2Util.validate_token(token, "1234")
+      [head | tail] = claims
+      assert elem(head, 1) == "Invalid token"
```

Comment:
is the `head` another array?

Reply:
The head is a tuple.

Reply:
could you do pattern matching to get the 1 element of the touple?

Reply:
Done.